python/pacemaker/_cts/network.py

```diff
@@ -0,0 +1,34 @@
+""" Network related utilities for CTS """
+
+__all__ = ["next_ip"]
+__copyright__ = "Copyright 2000-2023 the Pacemaker project contributors"
+__license__ = "GNU General Public License version 2 or later (GPLv2+) WITHOUT ANY WARRANTY"
+
```

Comment:
should this import anything?

Reply:
Not in its current form, doesn't look like.  When I make the changes as described below, maybe it'll need something.